src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/config/AuthHeaderWebFilter.kt

```diff
@@ -0,0 +1,19 @@
+package uk.gov.justice.digital.hmpps.hmppsallocations.config
+
+import org.springframework.http.HttpHeaders
+import org.springframework.stereotype.Component
+import org.springframework.web.server.ServerWebExchange
+import org.springframework.web.server.WebFilter
+import org.springframework.web.server.WebFilterChain
+import reactor.core.publisher.Mono
+
+@Component
+class AuthHeaderWebFilter : WebFilter {
```

Comment:
to allow the auth token to be passed through to ARN API, need to set it in the context here to reference later